patches/build-android-pylib-utils-device_dependencies.py.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/build/android/pylib/utils/device_dependencies.py b/build/android/pylib/utils/device_dependencies.py
+index ff78f9f04e6a0172fa49a990997790f75b2dfc49..f5b19de57efa54e15ddaa5106248383f1faf14f7 100644
+--- a/build/android/pylib/utils/device_dependencies.py
++++ b/build/android/pylib/utils/device_dependencies.py
+@@ -47,6 +47,7 @@ _EXCLUSIONS = [
+     re.compile(r'.*.pak\.info'),
+     re.compile(r'.*\.incremental\.json'),
+ ]
++_EXCLUSIONS += [re.compile(r'.*/monochrome_bundle_module_paks/locales/.*'), re.compile(r'.*/android_webview/locales/.*'), re.compile(r'.*/weblayer/locales/.*'), re.compile(r'.*/net/data.*'), re.compile(r'.*/resources/inspector/.*'), re.compile(r'.*/third_party/catapult/.*'), re.compile(r'.*/third_party/requests'), re.compile(r'.*/tools/android/avd/proto'),]
```

Comment:
These data files are not used in Brave tests, but are uploaded to test device, which takes time and occasionally causes timeout error.